mp/src/game/server/momentum/mom_triggers.cpp

```diff
@@ -1570,6 +1570,9 @@ void CNoGrenadesZone::OnStartTouch(CBaseEntity* pOther)
         return;
 
     pSticky->SetCanExplode(false);
```

Comment:
This bool should also probably be updated, if the fizzle type is airborne only you'll want to set CanExplode back to true when the sticky lands.

Reply:
You can do this in the sticky physics collide function